src/fire.js

```diff
@@ -0,0 +1,23 @@
+import firebase from 'firebase';
+
+// Made with guidance from :
+// https://www.youtube.com/watch?v=cFgoSrOui2M
+
+/**
+ * A config method to handle all the credentials to connect
+ * to our firebase project.
+ */
+const firebaseConfig = {
+  apiKey: 'AIzaSyCaVXsW8SaK3pNPZ4Igaf2o7ZatdwCBlp4',
+  authDomain: 'cooks-books-app.firebaseapp.com',
+  databaseURL: 'https://cooks-books-app-default-rtdb.firebaseio.com',
+  projectId: 'cooks-books-app',
+  storageBucket: 'cooks-books-app.appspot.com',
+  messagingSenderId: '5777053309',
+  appId: '1:5777053309:web:b5c9aa32a44773fca38314',
+  measurementId: 'G-JJN7SC1CT2',
+};
```

Comment:
@rileyhgrant You shouldn't commit the API key. We need to generate a new one and move the credentials to the `.env` file. I'll open a PR with the files necessary to do this.